From 8ae8c6bba601ce9ae8a3d54776ef934fb0a7986d Mon Sep 17 00:00:00 2001
From: Kumar Gala <kumar.gala@linaro.org>
Date: Wed, 11 Nov 2020 08:09:50 -0600
Subject: [PATCH] [AArch64] Fix build issue with aarch64-builtins.c on MacOS X
 host

When building w/clang on MacOS X we get the following error:

gcc/config/aarch64/aarch64-builtins.c:1231:3: error: expected expression
AARCH64_INIT_MEMTAG_BUILTINS_DECL (IRG, irg, irg, fntype);

Fix this by adding '-std=gnu++11' to the compile line.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>
---
 gcc/config/aarch64/t-aarch64 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/aarch64/t-aarch64 b/gcc/config/aarch64/t-aarch64
index 11d20b7be14..d4c3154cf40 100644
--- a/gcc/config/aarch64/t-aarch64
+++ b/gcc/config/aarch64/t-aarch64
@@ -41,7 +41,7 @@ aarch64-builtins.o: $(srcdir)/config/aarch64/aarch64-builtins.c $(CONFIG_H) \
   $(srcdir)/config/aarch64/aarch64-simd-builtins.def \
   $(srcdir)/config/aarch64/aarch64-simd-builtin-types.def \
   aarch64-builtin-iterators.h
-	$(COMPILER) -c $(ALL_COMPILERFLAGS) $(ALL_CPPFLAGS) $(INCLUDES) \
+	$(COMPILER) -std=gnu++11 -c $(ALL_COMPILERFLAGS) $(ALL_CPPFLAGS) $(INCLUDES) \
 		$(srcdir)/config/aarch64/aarch64-builtins.c
 
 aarch64-sve-builtins.o: $(srcdir)/config/aarch64/aarch64-sve-builtins.cc \
-- 
2.25.4

